parlai/mturk/tasks/acute_eval/README.md

```diff
@@ -38,10 +38,10 @@ This is a template of the expected format with the minimal expected fields:
     {
       "is_onboarding": false,
       "speakers_to_eval": ["first_modelname", "second_modelname"],
+      "dialogue_ids": [dialogue_1_id, dialogue_2_id],
```

Comment:
Hmm - why the change? Now the dialogue IDs are a bit disassociated in placement from the dialogues themselves

Reply:
i mostly made this change to help with readability later on - yes, it does disassociate the `id`s from the dialogues themselves, but I figured the tradeoff was better later on. however, happy to change back if you feel strongly

Reply:
I kind of prefer having the ids associated with the conversations, at the cost of splitting up a line of code over several visual lines later - but not a big deal